fastlane/lib/fastlane/command_line_handler.rb

```diff
@@ -5,7 +5,8 @@ class CommandLineHandler
     # @param [Array] args A hash of all options (e.g. --env NAME)
     def self.handle(args, options)
       lane_parameters = {} # the parameters we'll pass to the lane
-      platform_lane_info = [] # the part that's responsible for the lane/platform definition
+      platform_lane_info = args.slice!(0, 2) # the part that's responsible for the lane/platform definition
```

Comment:
what happens if we call just `fastlane lanename key1:value1 key2:value2`, without explicitly adding the platform (because the one we are using is defined as default)?

Reply:
if you will omit platform name at the command-line, seem that default platform name will be passed in here.

Reply:
I don't think so, see here below your additions:

Reply:
Ah, I understood your review.
I fixed my mistake at https://github.com/fastlane/fastlane/pull/11695/commits/b551fe3b279bfe580e16123dab88e457068c649f

Thank you.